README.md

```diff
@@ -1,61 +1,64 @@
-# bootstrap - [AngularJS](http://angularjs.org/) directives specific to [Bootstrap](http://getbootstrap.com)
+# UI Bootstrap - [AngularJS](http://angularjs.org/) directives specific to [Bootstrap](http://getbootstrap.com)
 
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/angular-ui/bootstrap?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-
-***
-
 [![Build Status](https://secure.travis-ci.org/angular-ui/bootstrap.svg)](http://travis-ci.org/angular-ui/bootstrap)
 [![devDependency Status](https://david-dm.org/angular-ui/bootstrap/dev-status.svg?branch=master)](https://david-dm.org/angular-ui/bootstrap#info=devDependencies)
 
-## IMPT - 2015 PLANS AND ANGULAR 1.3 SUPPORT
+### Table of Contents
+- [Demo](#demo)
+- [Installation](#installation)
+    - [Bower](#Install with Bower)
+    - [NuGet](#Install with NuGet)
+    - [Custom](#Custom build)
+    - [Manual](#Manual download)
+- [Support](#support)
+    - [FAQ](#faq)
+    - [Supported browsers](#supported-browsers)
```

Comment:
Yes. The ones without spaces are main headers, the ones with spaces indent and are sub headers.